sklearn/metrics/_plot/tests/test_roc_curve_display.py

```diff
@@ -24,12 +27,32 @@ def data():
     return X, y
 
 
+# This data always (with and without `drop_intermediate`)
```

Comment:
Interesting point. I think that we should indeed avoid. But let's revisit after the PR.

Reply:
Forgot about this, glad I wrote the note. Let me create an issue so we don't forget.